src/auth/auth.controller.ts

```diff
@@ -182,6 +185,20 @@ export class AuthController {
     return token;
   }
 
+  @Get('oidc-client')
+  async getOidcClient(
+    @Query() query: OidcClientRequest,
+  ): Promise<OidcClientResponse> {
+    const client = this.keyCloakService.getClient(
+      (query.type as ClientType) || ClientType.ADMIN,
+    );
```

Comment:
You must never return private "ADMIN" secrets, you should keep them safe and secure.

Reply:
so, I should return only clientId?